metrotranscom/doorway
# Feat/add help center content

# Pull Request Template

## Issue Overview
Adding content & translations for the Help Center
Apologies this is such a large PR, but most of it is just text copied in & translated from content and they will double check it once we get it on the dev env for them.

This PR addresses #issue



Outstanding Issues:
there are some/one issue that I'm running into that I haven't been able to troubleshoot. Not sure if it's due to my work or  #186 . But would like to get some review/comments started in the meantime. Thanks!
- [ ] **Translations don't work** - seeing these, altho all the translations are in...also the translations bar at the top doesn't show on these help pages. Guessing it's all related, but I haven't had a chance to dive in to it yet.
![image](https://github.com/metrotranscom/doorway/assets/12400715/1110c1d6-4de5-4060-9557-98d973096f01)
On my local env if I change a value in any  of the *-card.tsx files the page will reload with all the text instead of this:
![image](https://github.com/metrotranscom/doorway/assets/12400715/c2322347-d6a7-4020-bac7-0416e0a7fbde)

## How Can This Be Tested/Reviewed?
Do the usual `yarn install`, `yarn build`, and `yarn dev` for the Public site (get the backend going too) and verify that the content looks good and all the "zippy's" work correctly. 

Once this makes it to the dev environment, I'll have the Content team verify the actual text/content.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
